### PR TITLE
Add docs to tarball

### LINKED ISSUE #11257: Undo the duplication of #11256

### PR DESCRIPTION
**Pull request by mikedanese**
Fixes #11257 @erictune

### CODE REVIEW

File: build/common.sh [hunk 1]
```diff
@@ -770,6 +770,7 @@ function kube::release::package_full_tarball() {
   cp -R "${KUBE_ROOT}/third_party/htpasswd" "${release_stage}/third_party/htpasswd"
 
   cp -R "${KUBE_ROOT}/examples" "${release_stage}/"
+  cp -R "${KUBE_ROOT}/docs" "${release_stage}/"
```
Comment (bgrant0607) [inline build/common.sh]: Ah, that's why build/release.sh fails...

**Comment by bgrant0607** [comment]
Please rebase.

**Comment by bgrant0607** [comment]
@k8s-bot test this please

**Comment by bgrant0607** [comment]
cc @thockin

**Comment by thockin** [comment]
LGTm but you have to rebase this since I picked up some of the same changes.

**Comment by mikedanese** [comment]
rebased.

File: hack/test-update-storage-objects.sh [hunk 2]
```diff
@@ -93,7 +93,7 @@ startApiServer
 
 # Create a pod
 kube::log::status "Creating a pod"
-${KUBECTL} create -f examples/pod.yaml
+${KUBECTL} create -f docs/user-guide/pod.yaml
```
Comment (thockin) [inline hack/test-update-storage-objects.sh]: does this not run in shippable?

**Comment by thockin** [comment]
I don't have all the context, but LGTM.

@erictune

File: hack/test-update-storage-objects.sh (hunk 2 above)
Comment (bgrant0607) [inline hack/test-update-storage-objects.sh]: It does:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/shippable.yml#L43

pod.yaml was copied to both directories previously.

**Comment by bgrant0607** [comment]
LGTM

File: build/common.sh (hunk 1 above)
Comment (mikedanese) [inline build/common.sh]: also required for e2e tests which succeeded locally after the move but failed on jenkins which only has the tarball, not the full source.

**Comment by bgrant0607** [comment]
build/release.sh is broken for me at the moment. @mikedanese did you confirm that release.sh works with this change?

File: hack/test-update-storage-objects.sh (hunk 2 above)
Comment (mikedanese) [inline hack/test-update-storage-objects.sh]: yes it runs in shippable. pod.yaml is duplicated in both locations to alleviate build woes cause by #11239. this finishes the migration.

**Comment by mikedanese** [comment]
@bgrant0607 I will confirm.

**Comment by mikedanese** [comment]
@bgrant0607 this includes the fix for build/release.sh

**Comment by bgrant0607** [comment]
Awesome. LGTM

**Comment by bgrant0607** [comment]
all green

**Pull request merged**